responsibleai/vision/components/rai_vision_insights/spec.yaml

```diff
@@ -66,20 +73,22 @@ outputs:
   ux_json:
     type: path
 code: ../src
-environment: azureml://registries/azureml/environments/responsibleai-vision-ubuntu20.04-py38-cpu/versions/10
+environment: azureml://registries/azureml/environments/responsibleai-vision-ubuntu20.04-py38-cpu/versions/16
```

Comment:
I see 15 used in the above lines. Is this intentional?

Reply:
yes, this will be updated again anyway once the images are released